src/main/java/org/cryptomator/ui/updatereminder/UpdateReminderComponent.java

```diff
@@ -21,9 +22,10 @@ public interface UpdateReminderComponent {
 	Lazy<Scene> updateReminderScene();
 
 	Settings settings();
+	UpdateChecker updateChecker();
```

Comment:
note needed if the model (aka settings) is updated to already contain a `LocalDateTime`, since the updateChecker is just using the settings.